### PR TITLE
Folders 4

### PR DESCRIPTION
**Pull request by AliciadW**

### CODE REVIEW

File: src/components/Cards.js [hunk 1]
```diff
@@ -91,6 +27,10 @@ const CardItem = styled.li`
    list-style-type: none;
    width: 30%;
    margin-bottom: 10px;
+   margin-right: 15px;
+   :nth-child(3n) {
```
Comment (schalkventer) [inline src/components/Cards.js]: Can you confirm that this works? If not you might have to write it as `&:nth-child`?

File: src/components/Cards.js [hunk 2]
```diff
@@ -160,18 +120,22 @@ const Content = styled(CardContent)`
...
+  const questions = props.questionsPages;
+  const resources = props.resourcesPages;
+
   if (props.stacked) {
     return (
       <CardWrapper>
-        {cardData.map(card => (
-          <CardItemStacked>
-            <CardLink to={card.to}>
+        {foldersArray.map((card, index) => (
```
Comment (schalkventer) [inline src/components/Cards.js]: Maybe break this out into a `Card.jsx` file. Generally it is good practice to not have embedded logic in JS span more than one line. What do you think?

File: src/components/Cards.js [hunk 3]
```diff
@@ -180,25 +144,47 @@ function Cards(props) {
       </CardWrapper>
     )
   }
+
   return (
     <CardWrapper>
-      {NotStacked.map(card => (
-        <CardItem>
-          <CardLink to={card.to}>
+      {questions.map((card, index) => (
```
Comment (schalkventer) [inline src/components/Cards.js]: This means you can just reuse `Card.jsx` here as well.

File: src/components/Header.js [hunk 4]
```diff
@@ -110,24 +128,24 @@ class Header extends Component {
...
-          {/*<EmailIcon />*/}
-          {/*</HeaderButton>*/}
-          {/*<HeaderButton color="inherit">*/}
-          {/*<DeleteIcon />*/}
-          {/*</HeaderButton>*/}
-          {/*TODO: Only show when stories have been saved on landing page*/}
-          {/*<HeaderButton color="inherit">*/}
-            {/*<SettingsIcon/>*/}
-          {/*</HeaderButton>*/}
+          { titleStory &&
```
Comment (schalkventer) [inline src/components/Header.js]: Also perhaps break this into it's own component.

File: src/components/Cards.js (hunk 1 above)
Comment (AliciadW) [reply src/components/Cards.js]: This does work as is.

File: src/components/Cards.js (hunk 2 above)
Comment (AliciadW) [reply src/components/Cards.js]: Sure, I'll break it out into a separate file.

File: src/components/Cards.js (hunk 3 above)
Comment (AliciadW) [reply src/components/Cards.js]: Thanks :-)

**Pull request merged**